Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -124,16 +124,6 @@ export RGW_POOL_PREFIX=default
 
 1. Paste the above output from `create-external-cluster-resources.py` into your current shell to allow importing the source data.
 
-2. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
```

Comment:
Step 1 will setup the environment for the import script, right? I don't see how this step can be moved to the operator since the operator doesn't have that environment.

Reply:
Yaa, I see
In step one maybe we can create K8s secret with that data.